projects/packages/videopress/package.json

```diff
@@ -27,6 +27,7 @@
 		"@automattic/jetpack-webpack-config": "workspace:* || ^1.3",
 		"@babel/core": "7.18.13",
 		"@babel/preset-react": "7.18.6",
+		"@storybook/client-api": "6.5.10",
```

Comment:
To be able to use [useArgs](https://storybook.js.org/docs/react/addons/addons-api#useargs) and integrate/sync the `currentPage` prop from the component to the storybook controls.